performance/startup/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+import org.jetbrains.kotlin.gradle.plugin.mpp.NativeBuildType
+import org.jetbrains.kotlin.RunKotlinNativeTask
+
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+plugins {
+    id("benchmarking")
```

Comment:
There is also jvmRun in this plugin. It seems that now for this benchmark running process for jvm should fail with error on second run. It isn't critical, I think, it can be done later in separate review.